exchanges/ftx/ftx_test.go

```diff
@@ -0,0 +1,1007 @@
+	}
+	_, err := f.DeleteTriggerOrder("triggerOrder123")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetFills(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetFills(spotPair, "", time.Time{}, time.Time{})
+	if err != nil {
+		t.Error(err)
+	}
+	_, err = f.GetFills(spotPair, "", time.Unix(1559881511, 0), time.Unix(1559901511, 0))
+	if err != nil {
+		t.Error(err)
+	}
+	_, err = f.GetFills(spotPair, "", time.Unix(1559901511, 0), time.Unix(1559881511, 0))
+	if err == nil {
+		t.Error(err)
+	}
+}
+
+func TestGetFundingPayments(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetFundingPayments(time.Time{}, time.Time{}, "")
+	if err != nil {
+		t.Error(err)
+	}
+	_, err = f.GetFundingPayments(time.Unix(1559881511, 0), time.Unix(1559901511, 0), "")
+	if err != nil {
+		t.Error(err)
+	}
+	_, err = f.GetFundingPayments(time.Unix(1559901511, 0), time.Unix(1559881511, 0), "")
+	if err == nil {
+		t.Error(err)
+	}
+}
+
+func TestListLeveragedTokens(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLeveragedTokens()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetTokenInfo(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetTokenInfo("")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestListLTBalances(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLTBalances()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestListLTCreations(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLTCreations()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestRequestLTCreation(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.RequestLTCreation(testToken, 1)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestListLTRedemptions(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLTRedemptions(testToken, 5)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetQuoteRequests(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetQuoteRequests()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetYourQuoteRequests(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetYourQuoteRequests()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestCreateQuoteRequest(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.CreateQuoteRequest(strings.ToUpper(currency.BTC.String()), "call", order.Buy.Lower(), strconv.FormatInt(time.Now().AddDate(0, 0, 3).UnixNano()/1000000, 10), "", 0.1, 10, 5, 0, false)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestDeleteQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.DeleteQuote("testing123")
```

Comment:
This test also returned with `{"result":[],"success":true}` and errored

Reply:
question here: i got 404 for this cozz itssuppoesd to be numbers only.. how did u get success true :D. now im getting Request not found coz i dont have any quotes which is the expected output, so do u mind checking again when u get a chance? after i push up changes.

Reply:
This may have been in the wrong spot then. I also get a 404 now. I'll retest things and comment on any test like the scenario above